contracts/src/v0.8/keystone/CapabilityRegistry.sol

```diff
@@ -516,11 +523,15 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
     s_dons[id].id = id;
     s_dons[id].isPublic = isPublic;
 
+    ++s_dons[id].configCount;
+    uint32 configCount = s_dons[id].configCount;
```

Comment:
```suggestion
    uint32 configCount = 1; // Starting from 1 to avoid 0 cases
    s_dons[id].configCount = configCount; 
```

Reply:
To help with error cases, the actual storage value can be set at the very end. We are adding a new DON, so config count will always be 1.